kytos/core/interface.py

```diff
@@ -68,6 +72,22 @@ def __repr__(self):
         return f"TAG({self.tag_type!r}, {self.value!r})"
 
 
+# pylint: disable=super-init-not-called
+class TAGRange(TAG):
+    """Class that represents an User-to-Network Interface with
+     a tag value as a list."""
+
+    def __init__(
+        self,
+        tag_type: str,
+        value: list[list[int]],
+        mask_list: list[str, int]
```

Comment:
`mask_list` is going to store the `vlan/mask` or `vlan` lists that are created from `value`. For example:
```
value = [[104, 110]]
mask_list = ['104/4092', '108/4094', 110]
```
Is it okay to have it in TAGRange? `mask_list` is needed to send requests to `sdntrace_cp` and it was being calculated twice per request.

Reply:
> why not just just use `list[str]`
For more info, [comment](https://github.com/kytos-ng/mef_eline/pull/396#discussion_r1384185684). I am still looking into this but it does not interrupt any of the changes for `vlan_range`.

Reply:
Right. Let's keep it this way. Good catch.